File: ssr_pkg/src/yao_node.cpp
#include<ros/ros.h>
#include<std_msgs/String.h>

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "yao_node");
    printf("你已经成功编译了二个ROS节点\n");

    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("yao_node", 10);
    ros::Rate loop_rate(10);
    while(ros::ok())
    {
       printf("你已经成功编译了二个ROS节点！\n");
       std_msgs::String msg;
       msg.data = "happy, ROS!";
       pub.publish(msg);
       loop_rate.sleep();
    }
    return 0;
}

File: ssr_pkg/scripts/yao_node.py
#!/usr/bin/env python3
#coding=utf-8
import rospy
from std_msgs.msg import String

if __name__== "__main__":
  rospy.init_node('yao_node')
  rospy.logwarn("This is second warning message")

  pub = rospy.Publisher('yao_node', String, queue_size=10)

  rate = rospy.Rate(10) # 10hz

  while not rospy.is_shutdown():
    rospy.loginfo("This is second message")
    msg = String()
    msg.data = "Hello,ROS!"
    pub.publish(msg)
    rate.sleep()